src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -131,13 +139,20 @@ internal class HttpSource
 
                 using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
                 {
+                    if (!throwNotFound && response.StatusCode == HttpStatusCode.NotFound)
```

Comment:
Cool

-----Original Message-----
From: "David Fowler" <notifications@github.com>
Sent: ‎6/‎13/‎2015 11:15 AM
To: "aspnet/dnx" <dnx@noreply.github.com>
Cc: "Yishai Galatzer" <yishai_galatzer@yahoo.com>
Subject: Re: [dnx] Add NuGet v3 feed support (#2058)

In src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs:
> @@ -131,13 +139,20 @@ internal class HttpSource
>  
>                  using (var response = await _client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead))
>                  {
> +                    if (!throwNotFound && response.StatusCode == HttpStatusCode.NotFound)
We're doing the real change in release and dev
—
Reply to this email directly or view it on GitHub.